### PR TITLE
Html rules UI enhancement

### PR DESCRIPTION
**Pull request by tombui99**
- Renamed button and the title of the popup to 'Enabled Rule'
- Check all HTML Hint Rules by default
- Check HTML Hint Rules that were ran on the scan
- Fix the UX so it does not have an underline if there is not link
- Add a link to https://www.ssw.com.au/rules/avoid-clear-text-email-addresses-in-web-pages
- Move 'Enabled Rule' button from inside a scan to next to the URL

### CODE REVIEW

File: docker/utils.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ const { htmlHintConfig, fetchHtml, getHTMLHintRules } = require("./api");
 const R = require("ramda");
```
Comment (nvhoanganh) [inline docker/utils.js:8]: why is this changed? I don't see any changes apart from formatting changes

File: ui/src/components/htmlhintcomponents/UpdateHTMLRules.svelte [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<script>
+<script async>
```
Comment (nvhoanganh) [inline ui/src/components/htmlhintcomponents/UpdateHTMLRules.svelte:1]: why is this async?

Comment (nvhoanganh) [reply ui/src/components/htmlhintcomponents/UpdateHTMLRules.svelte:1]: don't see any await

File: docker/utils.js (hunk 1 above)
Comment (tombui99) [reply docker/utils.js:8]: Yes - they are only formatting changes
